cfme/fixtures/candu.py

```diff
@@ -73,7 +73,7 @@ def candu_db_restore(temp_appliance_extended_db):
     # get DB backup file
     db_storage_hostname = conf.cfme_data.bottlenecks.hostname
     db_storage_ssh = SSHClient(hostname=db_storage_hostname, **conf.credentials.bottlenecks)
-    rand_filename = "/tmp/db.backup_{}".format(fauxfactory.gen_alphanumeric())
+    rand_filename = f"/tmp/db.backup_{fauxfactory.gen_alphanumeric()}"
```

Comment:
```suggestion
rand_filename = fauxfactory.gen_alphanumeric(start="/tmp/db.backup_", length=20)
```

Reply:
If you're going to implement this then same change is required everywhere in this PR.

Reply:
@valaparthvi this change more related to pyupgrade. we are adding this in support of last changes. We will do in another PR; I'm avoiding this as inderectly fauxfactory also using `.format`